puzzle_simple_5.js

```diff
@@ -0,0 +1,172 @@
+/*
+@title: puzzle_simple_5
+@author: P. Rama Ganesh
+
```

Comment:
```suggestion
*/

```
You forgot to close the comment!